homeassistant/components/tibber/sensor.py

```diff
@@ -146,13 +152,25 @@ def _update_current_price(self):
             if now.date() == price_time.date():
                 max_price = max(max_price, price_total)
                 min_price = min(min_price, price_total)
+                if price_time.hour < 8:
+                    off_peak_1 = max(off_peak_1, price_total)
+                    num1 += 1
+                elif price_time.hour < 20:
+                    off_peak = max(off_peak, price_total)
+                    num0 += 1
+                else:
+                    off_peak_2 = max(off_peak_2, price_total)
+                    num2 += 1
                 num += 1
                 sum_price += price_total
         self._state = state
         self._device_state_attributes['max_price'] = max_price
         self._device_state_attributes['avg_price'] = round(sum_price / num, 3)
         self._device_state_attributes['min_price'] = min_price
         self._device_state_attributes['price_level'] = level
+        self._device_state_attributes['off_peak_1'] = off_peak_1 / num1
+        self._device_state_attributes['off_peak'] = off_peak / num0
+        self._device_state_attributes['off_peak_2'] = off_peak_2 / num2
```

Comment:
What are we calculating for these attributes?

Are we risking a division by zero error?